web/frontend/src/mocks/handlers.ts

```diff
@@ -214,15 +204,15 @@ export const handlers = [
       CHANGE_STATUS_TIMER
     );
 
-    return res(ctx.status(200), ctx.text('Action successfully done'));
+    return res(ctx.status(200), ctx.json({ Status: 0, Token: 'blabla' }));
```

Comment:
you should have a constant `const fakeToken = "fake token"` instead of those blablas